### ISSUE TITLE
[Bug]: wrapper breaks signature for normalize_min_max() in 0.8.3

### ISSUE THREAD

**Issue by dfpasdel** [kornia/kornia#3745]
### 🐛 Describe the bug

When updating to 0.8.3, my call to `normalize_min_max()` got broken, with error: `normalize_min_max() missing 1 required positional argument: 'input'`. However, my call to function seems to be correct according to documentation.

It seems that in 0.8.3, the function got wrapped by `perform_keep_shape_image`:

```
@perform_keep_shape_image
def normalize_min_max(x: torch.Tensor, min_val: float = 0.0, max_val: float = 1.0, eps: float = 1e-6) -> torch.Tensor:
```
where the wrapper uses `input` instead of `x`: `def _wrapper(input: torch.Tensor, *args: Any, **kwargs: Any) -> torch.Tensor:`

To me it looks like a bug, but maybe I am not using the package correctly.

Could someone help please? Thanks in advance.

### 🔄 Steps to Reproduce

```bash
Run minimal code example with 0.8.3
```

### 💻 Minimal Code Example

```python
import torch
import kornia
from kornia.enhance import normalize_min_max

print(kornia.__version__)

def not_working(inpt):
    try:
        output = normalize_min_max(x=inpt,   # According to documentation, but fails on 0.8.3
                                   min_val=0.0,
                                   max_val=1.0,
                                   eps=0.00001)
        print("Success")
    except Exception as e:
        print(e)

def working(inpt):
    try:
        output = normalize_min_max(input=inpt,  # Does not match with documentation, but works
                                   min_val=0.0,
                                   max_val=1.0,
                                   eps=0.00001)
        print("Success")
    except Exception as e:
        print(e)



if __name__ == '__main__':
    inpt = torch.zeros(1, 1, 112, 112)
    not_working(inpt)
    working(inpt)
```

### ✅ Expected behavior

Expect `not_working` function to not raise exception using 0.8.3

### ❌ Actual behavior

`not_working` function throws `normalize_min_max() missing 1 required positional argument: 'input'`

`working` function does not throws, but does not comply with the signature in the documentation

### 🔧 Environment

```shell
- Kornia version: 0.8.3
- PyTorch version: 2.8.0
- Python version: 3.12
- OS (e.g., Linux, macOS, Windows): Windows
- Installation method (pip, conda, from source): pip
```

### 🤝 Contribution Intent

- [ ] I plan to submit a PR to fix this bug
- [x] I'm reporting this bug but not planning to fix it

**Comment by Haris-bin-shakeel** [comment]
I've traced the bug: `@wraps(f)` is already applied in `perform_keep_shape_image`, which preserves metadata but not the runtime parameter binding. The wrapper's `_wrapper(input, *args, **kwargs)` signature takes precedence, so passing the tensor via a keyword argument like `x=` fails at runtime.

The fix would be to rename the first parameter of `normalize_min_max` (from `x` to `input`) and `jpeg_codec_differentiable` (from `image_rgb` to `input`). This aligns them with both the wrapper's hardcoded signature and PyTorch's own convention (the other 5 functions using this decorator already correctly use `input`). Happy to submit a PR if this can be assigned to me.

**Comment by Teja1631** [comment]
Hi, I'd like to work on this if it's still unassigned.

The root cause is clear: `@perform_keep_shape_image` defines `_wrapper(input, *args, **kwargs)`, so the wrapped function's first parameter name (`x` in `normalize_min_max`, `image_rgb` in `jpeg_codec_differentiable`) gets shadowed at runtime — passing `x=...` as a keyword fails because the wrapper only sees `input`.

Fix: rename the first parameter to `input` in both `normalize_min_max` and `jpeg_codec_differentiable` to align with the wrapper's hardcoded signature. I'll add a regression test and update the docstrings accordingly.

Could a maintainer assign this to me? Happy to submit a PR promptly.

**Comment by edgarriba** [comment]
feel free to send a fixing PR
